### ISSUE TITLE
Unable to install '../simple-markdown: Appears to be a git repo or submodule.'

### ISSUE THREAD

**Issue by zamiang** [Khan/simple-markdown#69]
```npm outdated
npm ERR! path /Users/bar/commons/node_modules/simple-markdown
npm ERR! code EISGIT
npm ERR! git /Users/foo/bar/node_modules/simple-markdown: Appears to be a git repo or submodule.
npm ERR! git     /Users/foo/bar/node_modules/simple-markdown
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

**Comment by ariabuckles** [comment]
Oh gosh sorry; I’ll look into this asap. In the meantime, `npm install simple-markdown@0.4.4` might work

**Comment by ariabuckles** [comment]
Fixed in `0.5.1`, sorry about that.

To update, run:

```
rm -rf ./node_modules/simple-markdown/.git  # or otherwise specify the path to simple-markdown
npm install simple-markdown@latest
```

Thanks for letting me know about this.

**Comment by zamiang** [comment]
Thanks!
